setup/artifacts/gcs.md

```diff
@@ -8,15 +8,15 @@ sidebar:
 {% include toc %}
 
 Spinnaker stages that read data from artifacts can consume
-[GCS](https://cloud.google.com/storage/) objects as artifacts.
+[GCS](https://cloud.google.com/storage/){:target="_blank"} objects as artifacts.
```

Comment:
Looks like as soon as you add this (on every page) the markdown is in italics until the end of the page, which seems odd to me...do you have any idea why that might be happening? (It just makes me wonder if there's a missing closing brace or something, even though I can't imagine where that would need to go.

Reply:
The `_` character starts italics

Reply:
Escaping it shouldn't work. That is, I would expect it to fix the displayed markdown in editor but break the link in the browser. But it does work.